### PR TITLE
Ocrvs 3126 - Form radio buttons should be Checkbox's

### PR DESCRIPTION
**Pull request by modupeadeonojobi**
https://user-images.githubusercontent.com/66411673/196004758-e961d261-46c6-47e6-9fdb-586ef4a574ee.mp4

### CODE REVIEW

**Comment by jpye-finch** [comment]
@modupeadeonojobi looks good to me!

**Comment by Zangetsu101** [comment]
@modupeadeonojobi Apologies about the confusion. You're right we need to change the conditions but the name of the variable had me confused tbh. Could we change it to `detailsUnavailable` or `detailsDontExist` to better convey the meaning?

**Comment by Zangetsu101** [comment]
@modupeadeonojobi Could you take a look at why the tests are failling please?

File: packages/client/src/forms/configuration/administrative/addresses.ts [hunk 1]
```diff
@@ -47,13 +47,13 @@ const fathersDetailsDontExistBasedOnContactAndInformant =
...
-  'draftData && draftData.mother && !draftData.mother.detailsExist'
+  'draftData && draftData.mother && !draftData.mother.detailsDontExist'
 
 // if mothers details do not exist
-const mothersDetailsDontExist = '!values.detailsExist'
+const mothersDetailsDontExist = 'values.detailsDontExist'
 
-// if fathers details do not exist
-const fathersDetailsDontExist = '!values.detailsExist'
+// if fathers details do not existdraftData.
```
Comment (Zangetsu101) [inline packages/client/src/forms/configuration/administrative/addresses.ts:55]:
```suggestion
// if fathers details do not exist
```

File: packages/client/src/forms/configuration/default.ts [hunk 2]
```diff
@@ -2239,7 +2217,7 @@ export const registerForms: IDefaultRegisterForms = {
                       'Hidden for Parent Details none or Mother only',
                     action: 'hide',
                     expression:
-                      'draftData && draftData.mother && !draftData.mother.detailsExist && !mothersDetailsExistBasedOnContactAndInformant'
+                      'draftData && draftData.mother && !draftData.mother.detailsDontExist && !mothersDetailsExistBasedOnContactAndInformant'
```
Comment (Zangetsu101) [inline packages/client/src/forms/configuration/default.ts:2220]:
```suggestion
                      'draftData && draftData.mother && draftData.mother.detailsDontExist && !mothersDetailsExistBasedOnContactAndInformant'
```

File: packages/client/src/forms/configuration/default.ts [hunk 3]
```diff
@@ -2283,7 +2261,7 @@ export const registerForms: IDefaultRegisterForms = {
                       'Hidden for Parent Details none or Father only',
                     action: 'hide',
                     expression:
-                      'draftData && draftData.father && !draftData.father.detailsExist && !fathersDetailsExistBasedOnContactAndInformant'
+                      'draftData && draftData.father && !draftData.father.detailsDontExist && !fathersDetailsExistBasedOnContactAndInformant'
```
Comment (Zangetsu101) [inline packages/client/src/forms/configuration/default.ts:2264]:
```suggestion
                      'draftData && draftData.father && draftData.father.detailsDontExist && !fathersDetailsExistBasedOnContactAndInformant'
```

File: packages/client/src/i18n/messages/form.ts [hunk 4]
```diff
@@ -162,7 +162,9 @@ interface IFormMessages
   fatherFirstNamesEng: MessageDescriptor
   fatherName: MessageDescriptor
   fathersDetailsExist: MessageDescriptor
+  fathersDetailsExistFieldLabel: MessageDescriptor
```
Comment (Zangetsu101) [inline packages/client/src/i18n/messages/form.ts:165]: Could we change this to `fathersDetailsDontExist`? Same for the mother's as well

**Comment by modupeadeonojobi** [comment]
> @modupeadeonojobi Could you take a look at why the tests are failling please?

Alright

File: packages/client/src/forms/configuration/administrative/addresses.ts [hunk 5]
```diff
@@ -47,13 +47,13 @@ const fathersDetailsDontExistBasedOnContactAndInformant =
 
 // if mothers details do not exist on other page
 const mothersDetailsDontExistOnOtherPage =
-  'draftData && draftData.mother && !draftData.mother.detailsExist'
+  'draftData && draftData.mother && !draftData.mother.detailsDontExist'
```
[the remark this answers is not in the document]
Comment (modupeadeonojobi) [reply packages/client/src/forms/configuration/administrative/addresses.ts:50]: Okay

File: packages/client/src/forms/configuration/administrative/addresses.ts (hunk 1 above)
Comment (modupeadeonojobi) [reply packages/client/src/forms/configuration/administrative/addresses.ts:55]: Okay

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**